docs/cost/ids/core.md

```diff
@@ -0,0 +1,271 @@
+# Core standard
+
+<style>
+.wy-nav-content {
+  max-width: 1200px;
+}
+</style>
+
+This page documents a mapping from the 'core' CoST IDS elements to OC4IDS fields and codes. The organization of the mapping tables reflects the structure of the CoST IDS.
+
+The CoST IDS sets out a number of disclosure requirements under the heading of 'information for disclosure upon request', also known as 'reactive disclosure'. You can disclose these elements proactively using OC4IDS. Separate tables are provided for project-level and process-level reactive disclosures.
```

Comment:
Should this paragraph also be present in `ocds.md`?

Reply:
Yes! Added in https://github.com/open-contracting/infrastructure/pull/422/commits/a198afb8f399543876b2b07ff0cd56481760c398